src/app.py

```diff
@@ -492,11 +492,11 @@ def matches(
 
 @app.get(
     "/api/job/{job_id}",
-    response_model=JobSchema,
+    response_model=Job,
```

Comment:
We're fine with returning the whole raw from db, right? I.e. there's no sensitive stuff there that users shouldn't have access to

Reply:
(Because if that was the case, we could define a schema for reading the job object)

Reply:
Hmm, for postgres migration I had to define `internal_id` for the Job object. It's not really sensitive, but by design it's not necessary for anything.